help/sources/tutorials/api/create/analytics/mixpanel.md

```diff
@@ -0,0 +1,1072 @@
+---
+keywords: Experience Platform;home;popular topics;sources;connectors;source connectors;sources sdk;sdk;SDK
+solution: Experience Platform
+title: Documentation self-service template
+topic-legacy: tutorial
+description: Learn how to connect Adobe Experience Platform to Mixpanel using the Flow Service API.
+hide: true
+hidefromtoc: true
+---
+# Create a *Mixpanel* source connection in the UI
+
+This tutorial provides steps for creating a *Mixpanel* source connector using the Platform user interface.
```

Comment:
Let's update this to: "The following tutorial walks you through the steps to create a source connection and a dataflow to bring Mixpanel data to Platform using the [Flow Service API](https://developer.adobe.com/experience-platform-apis/references/flow-service/).

Reply:
Oops sorry CPE.